src/common/devcard.ts

```diff
@@ -25,13 +25,19 @@ export const getMostReadTags = async (
   }));
 };
 
-const getFavoriteSourcesLogos = async (
+interface FavoriteSourcesProps {
+  alias: string;
+  limit: number;
+}
+
+export const generateFavoriteSourcesQuery = (
   con: DataSource,
   userId: string,
-): Promise<string[]> => {
-  const sources = await con
+  { alias = 'source', limit = 5 }: Partial<FavoriteSourcesProps> = {},
```

Comment:
What alternative `alias` do we use?

Reply:
I think its just when you want to modify the query since the function returns the query builder vs the data.

Reply:
Yes, this is to make it easier to add queries and avoid issues of not knowing the relevant entity. However, we will now close this PR.